playground-local/app.vue

```diff
@@ -10,31 +10,33 @@ const password = ref('')
 
 <template>
   <div>
-    <pre>Status: {{ status }}</pre>
+    <pre>Status: <span data-testid="status">{{ status }}</span></pre>
```

Comment:
Why do you name it `data-testid`? Would it make more sense to add another dash? -> `data-test-id`